darwin/future/core/client.py

```diff
@@ -12,7 +12,7 @@
 from darwin.future.core.types.common import QueryString
 from darwin.future.exceptions.client import NotFound, Unauthorized
 
-JSONType = Dict[str, Any]  # type: ignore
+JSONType = Union[Dict[str, Any], List[Dict[str, Any]]]  # type: ignore
```

Comment:
The JSON that is returned from the `workflows` endpoint is a JSON array of objects, so this needed updating.